src/main/java/mate/academy/hibernate/relations/service/impl/ActorServiceImpl.java

```diff
@@ -1,16 +1,25 @@
 package mate.academy.hibernate.relations.service.impl;
 
+import java.util.Optional;
+import mate.academy.hibernate.relations.dao.impl.ActorDaoImpl;
 import mate.academy.hibernate.relations.model.Actor;
 import mate.academy.hibernate.relations.service.ActorService;
 
 public class ActorServiceImpl implements ActorService {
+    private ActorDaoImpl actorDao;
+
+    public ActorServiceImpl(ActorDaoImpl actorDao) {
+        this.actorDao = actorDao;
+    }
+
     @Override
     public Actor add(Actor actor) {
-        return null;
+        return actorDao.add(actor);
     }
 
     @Override
     public Actor get(Long id) {
-        return null;
+        Optional<Actor> optionalActor = actorDao.get(id);
+        return optionalActor.get();
```

Comment:
Calling `get()` on an Optional without checking if it's present can cause a `NoSuchElementException`. It would be safer to use `orElseThrow()` with a meaningful exception, ensuring that if the `Actor` is not found, the service layer throws an understandable error.